knowledge_repo/app/static/css/pages/base.css

```diff
@@ -360,7 +420,18 @@ img {
 
 .label-primary,
 .label-default,
-.label-subscribed,
+.label-subscribed {
```

Comment:
what's the purpose of this change? If it's simply to change the color of the subscribed label, you can keep this all the same, then below it add:
```css
.label-subscribed {
  border-color: #00A699
}
```

Reply:
weird. I tried exactly this just and the button radius and coloring become totally different